applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -406,12 +407,13 @@ def visualization_preparation(nn_out, distances, dist_threshold):
     detection_scores = []
     detection_boxes = []
     colors = []
-
-    distance = np.amin(distances + np.identity(len(distances)) * 2., 0)
+    
+    distance = np.amin(distances + np.identity(len(distances)) * 1000, 0)
```

Comment:
If 1000 < dist_threshold then this method won't work properly. I suggest setting this to 2*dist_threshold to ensure working for all dist_threshold inputs.

Reply:
since the dist_threshold is mostly in [150,200] centimeters range, it won't be greater than 1000 but the code is more general and readable in your edition. modified, thanks